frontend/src/pages/Profile.js

```diff
@@ -9,36 +9,45 @@ import ProfileCurrentBookings from "../components/ProfileCurrentBookings"
 import ProfileBookingHistory from "../components/ProfileBookingHistory"
 import { useAuthenticatedUser } from "../hooks/useAuthenticatedUser"
 
-const Profile = () => {
+/**
+ * Reference to a booking from the Firestore.
+ * @typedef {{ data(): { user_id: string, check_in: require("firebase/firestore").Timestamp, check_out: require("firebase/firestore").Timestamp } }} Booking
+ */
+
+export default function Profile() {
   const [user, userMetadata] = useAuthenticatedUser()
+  /**
+   * @type {[Array<Booking>, React.Dispatch<Array<Booking>>]}
+   */
   const [bookings, setBookings] = useState(undefined)
-  // const [userData, setUserData] = useState(undefined)
+
+  /**
+   * Retrieves the current user bookings from firestore.
+   * @param {string} userId
+   */
+  const getBookings = (userId) => {
+    getDocs(
+      query(
+        collection(db, "bookings"),
+        where("user_id", "==", doc(db, "users", userId))
```

Comment:
What was the reason that `doc(db, "users", userId)` was used instead of just `userId`? 
It doesn't seem to be able to get any of the collection data with this change for me.

Reply:
Hmm -- that's interesting. Testing locally with the emulator doesn't work unless I do it in this way.

My understanding is that using `doc(db, "users", userId)` here will make the query against a [DocumentReference[(https://firebase.google.com/docs/reference/js/v8/firebase.firestore.DocumentReference) (i.e., the `/users/userId` reference) rather than a string literal of the user id.